drivers/wifi/nrf700x/osal/hw_if/hal/src/hal_api.c

```diff
@@ -1121,18 +1121,19 @@ struct wifi_nrf_hal_dev_ctx *wifi_nrf_hal_dev_add(struct wifi_nrf_hal_priv *hpri
 	wifi_nrf_osal_spinlock_init(hpriv->opriv,
 				    hal_dev_ctx->lock_rx);
 
-	hal_dev_ctx->rx_tasklet = wifi_nrf_osal_tasklet_alloc(hpriv->opriv);
+	hal_dev_ctx->event_tasklet = wifi_nrf_osal_tasklet_alloc(hpriv->opriv,
+		WIFI_NRF_TASKLET_TYPE_BH);
```

Comment:
Indentation

Reply:
Tried again, aligning to open brace vs. not exceeding 100 lines vs. using tabs as much as possible, its impossible to do this alignment anyother (AFAIK), so, closing this comment as checkpatch is passing.